### PR TITLE
Fixing conferences list

### PR DESCRIPTION
**Pull request by MvRemmerden**
Removing dead links, adding industry specific aggregators and updating explanations

### CODE REVIEW

**Comment by striglia** [comment]
+1 I also noticed a few of these links (Lanyrd in particular) no longer working

File: README.md [hunk 1]
```diff
@@ -93,24 +93,22 @@ Participation" depending on the conference.
...
-* [ConferenceList](https://conferencelist.co) (conferences only)
 * [Conference Radar](http://conferenceradar.com)
 * [Code And Talk](https://codeandtalk.com/conferences) (Conferences and CFPs)
 * [KDNuggets](https://www.kdnuggets.com/meetings/), primarily data-focused conferences
 * [IT CFP List](https://github.com/softwaremill/it-cfp-list) - List of IT conferences Call For Papers, changes are also published via Twitter [@ItCfpList](https://twitter.com/ItCfpList)
 * [CFP Land](https://www.cfpland.com)
 * [Techmeme Events List](https://www.techmeme.com/events)
 * [conferCal](http://www.confercal.com)
+* [Design Conferences](https://design-conferences-2019.com) UX, UI, Product & Interaction Design Conferences
+* [Marketing Conferences](https://marketing-conferences-2019.com) SEO, Social Media, Content, Growth, B2B & B2C Conferences
```
Comment (MvRemmerden) [inline README.md]:
```suggestion
```

**Review by MvRemmerden — COMMENTED** [review]
Already dead

File: README.md [hunk 2]
```diff
@@ -93,24 +93,21 @@ Participation" depending on the conference.
 * [Opensource.com Events Calendar](https://opensource.com/resources/conferences-and-events-monthly)
 * [Papercall.io](https://www.papercall.io/cfps), tech with a DevOps focus
 * [WikiCFP](http://wikicfp.com/cfp/), primarily academic and industry * conferences
-* [DevOps Conferences](http://devopsconferences.org/)
 * [confs.tech](https://confs.tech)
```
Comment (MvRemmerden) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -93,24 +93,21 @@ Participation" depending on the conference.
...
 * [Central US Community Tech Events](http://www.davidgiard.com/CentralUSEvents.aspx)
-* [Mozilla Tech CFPs](https://tchspk.rs/cfp) (all events welcome, not just Mozilla ones)
-* [Lanyrd](http://lanyrd.com/calls/) (Conferences and CFPs)
 * [CFP Time](https://cfptime.org)
-* [ConferenceList](https://conferencelist.co) (conferences only)
 * [Conference Radar](http://conferenceradar.com)
 * [Code And Talk](https://codeandtalk.com/conferences) (Conferences and CFPs)
```
Comment (MvRemmerden) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -93,24 +93,19 @@ Participation" depending on the conference.
...
 * [Central US Community Tech Events](http://www.davidgiard.com/CentralUSEvents.aspx)
-* [Mozilla Tech CFPs](https://tchspk.rs/cfp) (all events welcome, not just Mozilla ones)
-* [Lanyrd](http://lanyrd.com/calls/) (Conferences and CFPs)
 * [CFP Time](https://cfptime.org)
-* [ConferenceList](https://conferencelist.co) (conferences only)
 * [Conference Radar](http://conferenceradar.com)
-* [Code And Talk](https://codeandtalk.com/conferences) (Conferences and CFPs)
 * [KDNuggets](https://www.kdnuggets.com/meetings/), primarily data-focused conferences
 * [IT CFP List](https://github.com/softwaremill/it-cfp-list) - List of IT conferences Call For Papers, changes are also published via Twitter [@ItCfpList](https://twitter.com/ItCfpList)
 * [CFP Land](https://www.cfpland.com)
```
Comment (MvRemmerden) [inline README.md]:
```suggestion
* [CFP Land](https://www.cfpland.com) 
```

**Comment by MvRemmerden** [comment]
@vmbrasseur Would you mind either merging or closing this one?

**Comment by vmbrasseur** [comment]
Thanks for the PR and the nudge, @MvRemmerden.

**Pull request merged**